examples/multiphysics/3d-dam-break/Lethe_pyvista_tools.py

```diff
@@ -0,0 +1,95 @@
+#############################################################################
+"""Class of methods to post-process lethe results with pyvista"""
```

Comment:
I am not sure I understand. Is it an alternative code to post-process the dam break case? This is not very clear to me . Can you explain a bit more what this is doing and why this has been added to this example?

Reply:
Sorry, this file shouldn't be here. It stayed here when I switched branches. It was a mistake and I just deleted it.